native/package.json

```diff
@@ -7,13 +7,16 @@
     "start:integreat-test-cms": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' react-native start --reset-cache",
     "start:integreat": "cross-env BUILD_CONFIG_NAME='integreat' react-native start --reset-cache",
     "start:malte": "cross-env BUILD_CONFIG_NAME='malte' react-native start --reset-cache",
+    "start:aschaffenburg": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native start --reset-cache",
```

Comment:
Not sure if we rather should use a abbreviation for these commands to make them shorter. Not sure about whether there is a reasonable one though.

Reply:
changed it to abapp